packages/rspack/etc/api.md

```diff
@@ -5928,7 +5928,7 @@ export interface LoaderContext<OptionsType = {}> {
     // (undocumented)
     getOptions(schema?: any): OptionsType;
     // (undocumented)
-    getResolve(options: Resolve): (context: any, request: any, callback: any) => Promise<any>;
+    getResolve(options: Resolve): ((context: string, request: string, callback: ResolveCallback) => void) | ((context: string, request: string) => Promise<string | false | undefined>);
```

Comment:
@h-a-n-a PTAL ❤️

Reply:
According to webpack's type:
```ts
  getResolve(options?: ResolveOptionsWithDependencyType): {
    (
      context: string,
      request: string,
      callback: (
        err: null | ErrorWithDetail,
        res?: string | false,
        req?: ResolveRequest,
      ) => void,
    ): void
    (context: string, request: string): Promise<string>
  }
```
I was wondering when this would return `false`?

Reply:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/9d04a933-0ec7-4e58-8e33-a7c09d06e388">

https://github.com/web-infra-dev/rspack/blob/8f9db27586ebde35908fc0d1f57742b443fa0bb8/packages/rspack/src/Resolver.ts#L38

Reply:
make me confusing too 
https://github.com/web-infra-dev/rspack/blob/8f9db27586ebde35908fc0d1f57742b443fa0bb8/crates/rspack_binding_values/src/resolver.rs#L36-L37

Reply:
@h-a-n-a is it acceptable for being merged?

Reply:
I checked out the typings in enhanced-resolve. Looks good to me now.

Ref: https://github.com/webpack/enhanced-resolve/blob/72999caf002f6f7bb4624e65fdeb7ba980b11e24/types.d.ts#L957